src/CounterModule.hpp

```diff
@@ -216,8 +216,34 @@ void
     for (auto& b : bytes) {
       output << b;
     }
-    TLOG_DEBUG(5) << "Size of the message in bytes: " << output.str().size();
-    KafkaExport(kafka_address, output.str(), topicname);
+    TLOG_DEBUG(TLVL_WORK_STEPS) << "Size of the message in bytes: " << output.str().size();
+
+    // Split the message if it's too big
+    // Kafka doesn't let it go through when it's very close to the limit
+    int max_size = .99 * 1e6;
```

Comment:
Since this is an int, might be better as a constant `int max_size = 990000`

Reply:
This is correct but the idea was to read this value from a configuration parameter, which I didn't add for the sake of not making 'large' changes but will be there for the next release